.drone.yml

```diff
@@ -0,0 +1,7 @@
+pipeline:
+  build:
+    image: in1t3r/runit:v0.1
+    commands:
+      - cmake CMakeLists.txt
```

Comment:
This is wrong. It should be "cmake .", but preferably this would test an out-of-source build, i.e. "mkdir build; cd build; cmake ..".